### PR TITLE
Parse extended raag information

### PR DESCRIPTION
**Pull request by ManjotS**

### CODE REVIEW

File: api/controllers/shabads.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const allColumns = `v.ID, v.Gurmukhi, v.GurmukhiUni, v.Translations, v.PageNo AS
     v.SourceID as SourceID, s.ShabadID, v.FirstLetterStr, v.MainLetters, v.Visraam,
     v.FirstLetterEng, v.Transliterations, v.WriterID, w.WriterEnglish,
     w.WriterGurmukhi, w.WriterUnicode, v.RaagID, r.RaagGurmukhi,
-    r.RaagUnicode, r.RaagEnglish, r.RaagWithPage, r.StartID, r.EndID,
```
Comment (tsingh777) [inline api/controllers/shabads.js:12]: hope no one using API was relying on those columns.

File: api/lib/getJSON.js [hunk 2]
```diff
@@ -138,13 +138,22 @@ const getSource = shabad => ({
   pageNo: shabad.PageNo,
 });
 
+/* eslint-disable no-param-reassign */
```
Comment (tsingh777) [inline api/lib/getJSON.js:141]: 👎 can we follow a more [functional programing](https://www.youtube.com/watch?v=e-5obm1G_FY) approach? I know we have tech debt already so if you want to wait for fixing all of it in one go, we can follow up.

File: api/controllers/shabads.js (hunk 1 above)
Comment (ManjotS) [reply api/controllers/shabads.js:12]: if they were their use case is very very broken. Tarsem singh just told us this is from some very old version of STTM and has no use

File: api/lib/getJSON.js (hunk 2 above)
Comment (ManjotS) [reply api/lib/getJSON.js:141]: Let's start work on v3 this year and include typescript and some of that work. better integration of multisearch and graphql.

**Pull request merged**